share/golosd/config/config_witness.ini

```diff
@@ -0,0 +1,113 @@
+# Endpoint for P2P node to listen on
+# p2p-endpoint =
+
+# Maxmimum number of incoming connections on P2P endpoint
+# p2p-max-connections =
+
+# P2P nodes to connect to on startup (may specify multiple times)
+# seed-node =
+
+# Pairs of [BLOCK_NUM,BLOCK_ID] that should be enforced as checkpoints.
+# checkpoint =
+
+# Endpoint for websocket RPC to listen on
+# rpc-endpoint =
+
+# Endpoint for TLS websocket RPC to listen on
+# rpc-tls-endpoint =
+
+# The TLS certificate file for this server
+# server-pem =
+
+# Password for this certificate
+# server-pem-password =
+
+# Block signing key to use for init witnesses, overrides genesis file
+# dbg-init-key =
+
+# API user specification, may be specified multiple times
+# api-user =
+
+webserver-thread-pool-size = 2
+webserver-http-endpoint = 0.0.0.0:8090
+webserver-ws-endpoint = 0.0.0.0:8091
+rpc-endpoint = 0.0.0.0:2001
+
+shared-file-size = 128G
+
+plugin = chain p2p json_rpc webserver network_broadcast_api witness
```

Comment:
`json_rpc` and `webserver` are used for commutations with clients, but there is no API plugins. May be add database_api for basic API functions without additional indexes? What do you think?